apps/system/test/unit/lockscreen/lockscreen_state_manager_test.js

```diff
@@ -155,7 +153,11 @@ suite('system/LockScreenStateManager', function() {
         return true;
       });
       subject.transfer({}).then(() => {
-        assert.isTrue(stubTransferOut.called, 'the method isn\'t called');
+        assert.isTrue(stubTransferTo.called,
+          'the |transferTo| method isn\'t called');
+        assert.isTrue(stubTransferOut.called,
+          'the |transferOut| method isn\'t called');
+        done();
       }).catch(done);
```

Comment:
1. I prefer to express error catching with an explicit `catch`, which for me is a bit more clear than put it as the second parameter of `then`. Although people use Promise should all know the meaning of second parameter.

Anyway I will modify that as you said.

2. Thanks for your reminder. I'll use `called` instead of `isTrue`.